ta4j-core/src/main/java/org/ta4j/core/num/Num.java

```diff
@@ -260,18 +260,4 @@ default float floatValue(){
         return getDelegate().floatValue();
     }
 
-
-    @Override
-    int hashCode();
-
-    @Override
-    String toString();
-
-    /**
-     * {@inheritDoc}
-     * @apiNote: This method should return true if `this` and `obj` are both NaN.
-     */
-    @Override
-    boolean equals(Object obj);
-
```

Comment:
I think forcing the implementing class to override the `hashCode()` method is necessary, because we also override the `equals()` method. It is also a consistent approach to have an explicit `toString()` method that converts the `Num` implementation to a human friendly string.